app/components/App/App.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { autobind } from 'core-decorators';
```

Comment:
If you agree that trailing commas are a good idea, you can use them in named imports too.

Reply:
done